pytest_selenium/drivers/firefox.py

```diff
@@ -47,6 +46,11 @@ def firefox_driver(request, capabilities, driver_path, firefox_profile):
 
 
 @pytest.fixture
+def firefox_path(request):
+    return request.config.getoption('firefox-path')
```

Comment:
This should be `firefox_path` rather than `firefox-path`, though I suspect both might work.